.travis.yml

```diff
@@ -18,8 +18,8 @@ sudo: required
 addons:
   postgresql: "9.3"
 before_install:
-  - wget "https://github.com/Masterminds/glide/releases/download/0.8.1/glide-0.8.1-linux-amd64.tar.gz"
-  - sudo tar -vxz -C /usr/local/bin --strip=1 -f glide-0.8.1-linux-amd64.tar.gz
+  - wget "https://github.com/Masterminds/glide/releases/download/0.8.3/glide-0.8.3-linux-amd64.tar.gz"
+  - sudo tar -vxz -C /usr/local/bin --strip=1 -f glide-0.8.3-linux-amd64.tar.gz
```

Comment:
Couldn't get it installing glide.yaml without it. Potentially because I have `0.8.3` locally, it could have messed that up but reverting glide to `0.8.1` seemed like an overkill